### ISSUE TITLE
API testing service

### ISSUE THREAD

**Issue by jimaek** [jsdelivr/globalping#210]
Lets consider using something like https://stepci.com/ to ensure that new PRs dont break the API.

**Comment by MartinKolarik** [comment]
That should be already covered by the tests we have but after #168 we might look into testing the API based on the yaml spec.

**Comment by mishushakov** [comment]
Hey
We have pushed some cool updates, such as Performance Testing, OpenAPI Import and gRPC support since the issue was opened

Maybe you could take a fresh look on [our website](https://stepci.com)?

Let me know if you have any questions or need any help! I'd be glad to support!
Thanks
